src/Schedule.h

```diff
@@ -93,26 +110,34 @@ enum class PrefetchBoundStrategy {
  * function is done by generating a loop nest that spans its
  * dimensions. We schedule the inputs to that function by
  * recursively injecting realizations for them at particular sites
- * in this loop nest. A LoopLevel identifies such a site. */
+ * in this loop nest. A LoopLevel identifies such a site. The site
+ * can either be a specific loopness within all stages of a function
```

Comment:
I'm not sure what "specific loopness" is. Is it meant to be "loop nest" or does this mean "a loop over over the same dimension across all stages of a function"? ("Loop nest" would imply a set of loops all of which are on the same var or dim I think, but it isn't quite clear to me even if that is the wording.)